File: p8/1/num.cpp
#include <iostream>
#include <cmath>

int main (int argc, char* argv[])
{
  double seno = 0;
  double coseno = 0;
  double tangente = 0;
  double logaritmo = 0;
  double raizc = 0;
  for(double i = 0; i < 30000000; i++)
  {
    seno += sin(i);
    coseno += cos(i);
    tangente += tan(i);
    logaritmo += log10(i);
    raizc += sqrt(i);
  }
  return 0;
}

File: p8/1/num.java
import java.util.*;
import java.lang.Math;

public class num {
public static void main(String args[])
{
  double seno = 0;
  double coseno = 0;
  double tangente = 0;
  double logaritmo = 0;
  double raizc = 0;
  for (double i = 0; i < 30000000; i++)
  { 
    seno += Math.sin(i);
    coseno += Math.cos(i);
    tangente += Math.tan(i);
    logaritmo += Math.log10(i);
    raizc += Math.sqrt(i);
  }
}

}
